src/Compilers/VisualBasic/Portable/Syntax/SyntaxKind.vb

```diff
@@ -137,7 +137,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         NamespaceBlock = 48                      ' NamespaceBlockSyntax : DeclarationStatementSyntax : StatementSyntax
         ''' <summary>
         ''' Represents the beginning statement of a namespace declaration. This node always
-        ''' appears as the Begin of a BlockStatement with Kind=NamespaceBlock.
+        ''' appears as the begin of a BlockStatement with Kind=NamespaceBlock.
```

Comment:
@CyrusNajmabadi Thanks a lot for reviewing and helping.
I'm having trouble finding a class named (or containing) "BlockStatement". Can you please point me to it?

Thanks again for helping.

Reply:
Pinging @CyrusNajmabadi. I'm stuck finding the correct cref for these "Begin"s. It's much more appropriate to use crefs IMO rather than reverting back to capitalized "Begin".

Reply:
I believe Cyrus is referring to things like http://sourceroslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Syntax/MethodBlockBaseSyntax.vb,ef83120e74640ac7.

Reply:
> 
> 
> I believe Cyrus is referring to things like http://sourceroslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Syntax/MethodBlockBaseSyntax.vb,ef83120e74640ac7.

@333fred I think this one will be applicable to comments that says "with Kind=MethodDeclarationBlock". But this specific one is NamespaceBlock and I couldn't find out the "Begin" it should be linked to.

Reply:
It looks like the NamespaceBlock should be http://sourceroslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Generated/Syntax.xml.Syntax.Generated.vb,af7b4106d18e493c,references.